### ISSUE TITLE
NameError: name 'cfg' is not defined when upgrading to 5.0.0

### ISSUE THREAD

**Issue by saultdon** [semiautomaticgit/SemiAutomaticClassificationPlugin#7]
Getting the following warning when upgraded to version 5.0.0
Seems to just be a warning, and it actually upgraded and installed the latest version.

```
NameError: name 'cfg' is not defined 
Traceback (most recent call last):
  File "/usr/share/qgis/python/qgis/utils.py", line 333, in startPlugin
    plugins[packageName] = package.classFactory(iface)
  File "/home/saultdon/.qgis2/python/plugins/SemiAutomaticClassificationPlugin/__init__.py", line 65, in classFactory
    from semiautomaticclassificationplugin import SemiAutomaticClassificationPlugin
  File "/usr/share/qgis/python/qgis/utils.py", line 607, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "/home/saultdon/.qgis2/python/plugins/SemiAutomaticClassificationPlugin/semiautomaticclassificationplugin.py", line 138, in 
    cfg.scipyCheck = "No"
NameError: name 'cfg' is not defined
```

I have python2-scipy 0.18.0 installed on Arch Linux.

**Comment by semiautomaticgit** [comment]
Thank you for reporting this.
I think that I solved this issue in version 5.0.1.
Please test this when it is available from the QGIS Plugin Manager.

Cheers

**Comment by semiautomaticgit** [comment]
Thank you.
Fixed this in v. 5.0.2

Cheers
